### DISCUSSION TITLE
API on

### DISCUSSION THREAD

**Discussion by willyantomaz** [joaoagr1/rpg-manager-system-api-java#4] [Q&A] [answered]
Qual servidor vc pretende hospedar essa aplicação ?

**Comment by joaoagr1** [comment, marked as answer]
Atualmente estou hospedando no Railway pela facilidade, mas pretendo migrar para serviços da AWS futuramente, pela alta disponibilidade e segurança.

**Reply by willyantomaz** [reply]
legal, espero que de certo.
